Sources/MessagingInApp/Views/InAppMessageView.swift

```diff
@@ -38,6 +44,9 @@ public class InAppMessageView: UIView {
         }
     }
 
+    // Delegate to handle custom action button tap.
+    public var inlineMessageDelegate: InlineMessageDelegate?
```

Comment:
```suggestion
    public weak var inlineMessageDelegate: InlineMessageDelegate?
```

Delegates are commonly `weak` in iOS development. I consider this a requirement for this feature as well.